modules/msf_integration.py

```
@@ -0,0 +1,410 @@
+            
+            if result.returncode != 0:
+                error_msg = result.stderr.decode('utf-8', errors='ignore')
+                raise RuntimeError(f"MSF载荷生成失败: {error_msg}")
+            
+            payload_data = result.stdout
+            command_line = ' '.join(cmd)
+            
+            return payload_data, command_line
+            
+        except subprocess.TimeoutExpired:
+            raise RuntimeError("MSF载荷生成超时")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)